NitroxClient/MonoBehaviours/Gui/InGame/PlayerKickedModal.cs

```diff
@@ -45,7 +45,7 @@ private static void InitSubWindow()
                 GameObject header = playerKickedSubWindow.FindChild("Header"); //Message Object
 
                 Text messageText = header.GetComponent<Text>();
-                messageText.text = "You have been kicked from the server";
+                messageText.text = reason;
```

Comment:
Might be good idea to set a default text if packet message was null or empty.
```suggestion
                messageText.text = string.IsNullOrWhiteSpace(reason) ? "You've been kicked from the server" : reason;
```